responsemanager/hooks/requesthook.go

```diff
@@ -82,6 +87,10 @@ func (irh *IncomingRequestHooks) Register(hook graphsync.OnIncomingRequestHook)
 	return graphsync.UnregisterHookFunc(irh.pubSub.Subscribe(hook))
 }
 
+type QueuedRequestResult struct {
```

Comment:
It's also ok to just remove this struct and return just the new context if that is all that's needed

Reply:
:thumbsup: